apps/issuer/models.py

```diff
@@ -672,6 +672,31 @@ def get_user_or_none(recipient_id, recipient_type):
 
     return user
 
+class IssuerStaffRequest(BaseVersionedEntity):
+    class Status(models.TextChoices):
+        PENDING = 'Pending', 'Pending'
+        APPROVED = 'Approved', 'Approved'
+        REJECTED = 'Rejected', 'Rejected'
+        REVOKED = 'Revoked', 'Revoked'
+
+    issuer = models.ForeignKey(Issuer, blank=False, null=False,
+                               on_delete=models.CASCADE, related_name='staffrequests')
+    user = models.ForeignKey('badgeuser.BadgeUser', blank=True, null=True, on_delete=models.SET_NULL)
```

Comment:
Shouldn't this not also be on_delete=models.CASCADE?
When the user is deleted, there is no need to keep the IssuerStaffRequest right?